### DISCUSSION TITLE
mdnotes invalid zotero 6.0

### DISCUSSION THREAD

**Discussion by nefesh1-almagor1** [argenos/zotero-mdnotes#160] [Q&A] [answered]
17 de marzo de 2022, la instalacion del plugin zotero-mdnotes es invalido en la aplicacion zotero version 6.0 alguien con el mismo problema?? como lo resuelvo

**Comment by pleo624** [comment]
Zotero just updated, and I am seeing the same thing! Is there a fix for this? Thanks!

**Comment by argenos** [comment, marked as answer]
Please see #133 for news on the update.
